packages/orbit-components/.storybook/orbitDecorator.tsx

```diff
@@ -1,13 +1,16 @@
 import * as React from "react";
 import jsxToString from "react-element-to-jsx-string";
-import { Source } from "@storybook/blocks";
+import { Highlight, themes } from "prism-react-renderer";
 
 import defaultTheme from "../src/defaultTheme";
 import OrbitProvider from "../src/OrbitProvider";
 import Heading from "../src/Heading";
 import Text from "../src/Text";
+import Tile from "../src/Tile";
 
 const OrbitDecorator = (storyFn, context) => {
```

Comment:
Can we get the types of these args from storybook and avoid the warnings? :thinking:

Reply:
I found `StoryFn` and `StoryContext` types in `@storybook/react` but tbh I am not 100% sure here.

If I add them here

```
const OrbitDecorator = (storyFn: StoryFn, context: StoryContext) => {
```

I need to change line 33 to
```
  const children = storyFn(null, context);
```
as `storyFn` requires two arguments.

I mean, there are no warnings anymore, everything works, but I didn't find usage of this "storyFn" function we have in the SB docs :( Any ideas?

Reply:
Ok, actually, importing `Decorator` type from `@storybook/react` should be solving this without any additional changed 🤔 

```
const OrbitDecorator: Decorator = (storyFn, context) => {
```